examples/regional_private_node_pool_oauth_scopes/outputs.tf

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "cluster_name" {
+  description = "Cluster name"
+  value       = "module.gke.name"
```

Comment:
Sorry, to clarify, the values do not need to be strings at all. They can just be direct references to attributes, like `value = module.gke.name`.

Reply:
My bad. Thank you for pointing that out. 👍